test/e2e/lib/components/secure-payment-component.js

```diff
@@ -76,6 +78,19 @@ export default class SecurePaymentComponent extends AsyncBaseContainer {
 		// SecurePaymentComponent.completeTaxDetailsInContactSection.
 		await this.completeTaxDetailsForCreditCard( { cardPostCode, cardCountryCode } );
 
+		const creditCardHandleSelector = By.css( 'label[for="card"]' );
+		if ( currentScreenSize() === 'mobile' ) {
+			await driverHelper.scrollIntoView( this.driver, creditCardHandleSelector );
+		}
+
+		// Sometimes the credit card form will be closed and it will require a click to be opened.
+		// This can happen when users have a credit card already associated with their account.
+		await driverHelper.selectElementByText(
+			this.driver,
+			creditCardHandleSelector,
+			'Credit or debit card'
+		);
+
```

Comment:
I noticed that a test didn't pass because it got stuck on the entering credit card details step. It happened because the testing user already had some credit cards stored so the credit card form was closed/hidden. To complete the test, I had to make the client to search for the credit card row, click it (to open the form), and then make it fill in the data.